src/routes/notification-page/index.jsx

```diff
@@ -0,0 +1,11 @@
+import Notifications from '../../components/notifications/notifications';
+
+const NotificationPage = () => {
+  return (
+    <div>
+      <Notifications />
+    </div>
```

Comment:
Any specific reason for wrapping it in a `div`, as I can see no classes on it?

```suggestion
     <Notifications />
```